app/models/user.rb

```diff
@@ -10,4 +14,10 @@ class User < ApplicationRecord
     :user,
     :admin
   ]
+
+  private
+
+  def slugify_user
+    self.slug = "#{self.name}-#{rand(1000..9999)}"
```

Comment:
```suggestion
    self.slug = "#{self.name}-#{rand(1000..9999)}".downcase.parameterize
```

Reply:
Acho que daria pra trocar esse `rand` pra alguma coisa que seja mais segura, talvez algo do tipo:
```rb
SecureRandom.hex(3)
```

Daí ficaria:

```suggestion
    self.slug = "#{self.name}-#{SecureRandom.hex(3)}".downcase.parameterize
```